policy/custom/definitions/policy/LA-Microsoft.AzureRecoveryVault-Backup/azurepolicy.rules.json

```diff
@@ -0,0 +1,175 @@
+{
+  "if": {
+    "allOf": [
+      {
+        "field": "type",
+        "equals": "Microsoft.RecoveryServices/vaults"
+      },
+      {
+        "field": "location",
+        "in": "[parameters('AzureRegions')]"
+      }
+    ]
+  },
+  "then": {
+    "effect": "deployIfNotExists",
+    "details": {
+      "type": "Microsoft.Insights/diagnosticSettings",
+      "existenceCondition": {
+        "allOf": [
+          {
+            "count": {
+              "field": "Microsoft.Insights/diagnosticSettings/logs[*]",
+              "where": {
+                "allOf": [
+                  {
+                    "field": "Microsoft.Insights/diagnosticSettings/logs[*].Category",
+                    "in": [
+                      "CoreAzureBackup",
+                      "AddonAzureBackupJobs",
+                      "AddonAzureBackupAlerts",
+                      "AddonAzureBackupPolicy",
+                      "AddonAzureBackupStorage",
+                      "AddonAzureBackupProtectedInstance"
+                    ]
+                  },
+                  {
+                    "field": "Microsoft.Insights/diagnosticSettings/logs[*].Enabled",
+                    "equals": "True"
+                  }
+                ]
+              }
+            },
+            "Equals": 6
+          },
+          {
+            "field": "Microsoft.Insights/diagnosticSettings/workspaceId",
+            "matchInsensitively": "[parameters('logAnalytics')]"
+          },
+          {
+            "field": "Microsoft.Insights/diagnosticSettings/logAnalyticsDestinationType",
+            "Equals": "Dedicated"
+          }
+        ]
+      },
+      "roleDefinitionIds": [
+        "/providers/Microsoft.Authorization/roleDefinitions/92aaf0da-9dab-42b6-94a3-d43ce8d16293",
+        "/providers/Microsoft.Authorization/roleDefinitions/b24988ac-6180-42a0-ab88-20f7382dd24c"
```

Comment:
The policy defines the Contributor role with `/providers/Microsoft.Authorization/roleDefinitions/b24988ac-6180-42a0-ab88-20f7382dd24c`.  This may not be needed.

The built in policy `Deploy Diagnostic Settings for Recovery Services Vault to Log Analytics workspace for resource specific categories.` only needs these 2 permissions:

  * Log Analytics Contributor - /providers/microsoft.authorization/roleDefinitions/92aaf0da-9dab-42b6-94a3-d43ce8d16293
  * Monitoring Contributor - /providers/microsoft.authorization/roleDefinitions/749f88d5-cbae-40b8-bcfc-e573ddc772fa

Can you update the role definitions to these 2?